src/app/components/audio-recordings/components/analyses-download/analysis-download-row.component.scss

```diff
@@ -0,0 +1,49 @@
+$vertical-line-color: grey;
+$margin-offset: 13px;
+
+.directory-listing-container {
+  display: grid;
+  grid-template-columns:
+    1fr
+    14rem
+    9rem
```

Comment:
This can absolutely be done.

I originally decided not to because the column widths would change when opening and closing nodes / items. This is because they were scaled to fit the file names, size, etc.. so if a file with a larger name / size came along, the entire table would re-scale to fit the content best.

Reply:
hmm you make a good point.

If you could max-width the name column and auto-min-width the two end columns (without reflow on folder open/close) then that would be good. Otherwise, yeah avoiding reflow take priority.